scripts/install-deps.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+echo "deb http://packages.laboratoriopublico.org/publiclab/ stretch main" > /etc/apt/sources.list.d/publiclab.list
+
+# Obtain key
+mkdir -p ~/.gnupg
+echo "disable-ipv6" >> ~/.gnupg/dirmngr.conf
```

Comment:
I realize this comes from Dockerfile but I wonder if line 8 would be enough (ipv4.pool...).

Reply:
What me to drop it and test it out? @icarito